frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -313,92 +318,96 @@ export function AnsibleAutomationsForm(props: {
-                                    setEditAnsibleJobs({ jobs: destroyPreJobs, setJobs: setDestroyPreJobs })
-                                    setEditAnsibleJob(ansibleJob)
-                                },
-                                onCreate: () => {
-                                    setEditAnsibleJobs({ jobs: destroyPreJobs, setJobs: setDestroyPreJobs })
-                                    setEditAnsibleJob({ name: '', extra_vars: {} })
-                                },
-                            },
-                            {
-                                id: 'destroyPostJob',
-                                type: 'OrderedItems',
-                                label: t('template.postDestroy.label'),
-                                placeholder: t('template.job.placeholder'),
-                                value: destroyPostJobs,
-                                onChange: setDestroyPostJobs,
-                                keyFn: (ansibleJob: AnsibleJob) => ansibleJob.name,
-                                cellsFn,
-                                onEdit: (ansibleJob) => {
-                                    setEditAnsibleJobs({ jobs: destroyPostJobs, setJobs: setDestroyPostJobs })
-                                    setEditAnsibleJob(ansibleJob)
-                                },
-                                onCreate: () => {
-                                    setEditAnsibleJobs({ jobs: destroyPostJobs, setJobs: setDestroyPostJobs })
-                                    setEditAnsibleJob({ name: '', extra_vars: {} })
-                                },
-                            },
-                        ],
-                    },
+                    ...(automationFeatureGateFlag
+                        ? ([
+                              {
+                                  type: 'Section',
+                                  title: t('template.create.scale'),
+                                  wizardTitle: t('template.create.scale.wizard.title'),
+                                  inputs: [
+                                      {
+                                          id: 'scalePreJob',
+                                          type: 'OrderedItems',
+                                          label: t('template.preScale.label'),
+                                          placeholder: t('template.job.placeholder'),
+                                          value: scalePreJobs,
+                                          onChange: setScalePreJobs,
+                                          keyFn: (ansibleJob: AnsibleJob) => ansibleJob.name,
+                                          cellsFn,
+                                          onEdit: (ansibleJob) => {
+                                              setEditAnsibleJobs({ jobs: scalePreJobs, setJobs: setScalePreJobs })
+                                              setEditAnsibleJob(ansibleJob)
```

Comment:
Not really related to these changes, but I would find this plural distinction hard to understand..